### ISSUE TITLE
jconfig.h.meson file missing

### ISSUE THREAD

**Issue by fzwoch** [mesonbuild/libjpeg#3]
```
Executing subproject libjpeg.

Project name: libjpeg
Native C compiler: cc (gcc 7.3.0 "cc (Debian 7.3.0-11) 7.3.0")
Has header "locale.h": YES

subprojects/jpeg-9a/meson.build:77:0: ERROR: File jconfig.h.meson does not exist.

```

From the `meson.build` file:

```
configure_file(input : 'jconfig.h.meson',
  output : 'jconfig.h',
configuration : cdata)
```

I can't seem to figure out where that `jconfig.h.meson` file is supposed to come from?

**Comment by jpakkane** [comment]
Probably someone (i.e. me) forgot to add that file to Git before doing the MR...

**Comment by sarum9in** [comment]
Closed by #4.
